deployer/scripts/install.sh

```diff
@@ -122,22 +122,18 @@ function generate_support_objects() {
 
   oc new-app logging-support-template
   kibana_keys=""; [ -e "$dir/kibana.crt" ] && kibana_keys="--cert=$dir/kibana.crt --key=$dir/kibana.key"
-  # use provided ca cert, if any, otherwise, use the internal ca cert
+  # use provided ca cert, if any
   if [ -e "$dir/kibana.ca.crt" ] ; then
       kibana_keys="$kibana_keys --ca-cert=$dir/kibana.ca.crt"
-  else
-      kibana_keys="$kibana_keys --ca-cert=$dir/ca.crt"
```

Comment:
if kibana.ca.crt is not provided, are kibana.crt and kibana.key still available?  If so, which CA signed kibana.crt?